ComparisonWithOrleans.md

```diff
@@ -0,0 +1,203 @@
+  
+  * Akka actors are explicitly created by their parent, implementing mandatory parental supervision. This allows precise control over when initialization actions are performed and which exact actor class is being created.
+  
+#### Virtual Actor Space
+
+  * In Orleans each type of Grain corresponds to a practically infinite space of Grain instances that conceptually all exist from the beginning of the universe to its end. The relation to the physical Actors that implement the Grains is explained similar to virtual and physical memory, but this comparison is misleading since the virtual address space of a process is explicitly populated with the desired contents instead of containing the whole system’s information by default.
+  
+  * Akka’s explicit lifecycle management requires that all currently running Actors must have been explicitly started in the past and will have to be explicitly stopped in the future. Higher-level abstractions like ClusterSharding provide the ability to opt into an Orleans-like model where instances can be addressed by their logical name instead of a concrete ActorRef, but the API for accessing them is provided by other Actors that perform the necessary lookups and initialization on behalf of the user.
+
+#### Automatic Actor Placement and Load Balancing
+
+  * Orleans Grains are deployed on silos that can span multiple cluster nodes and the user has no influence on their precise placement or load-dependent movement. This means that making use of elastically provisioned resources is fully automatic and built into the model.
+  
+  * Elastic deployment and load balancing are features that Akka users opt into by using ClusterSharding or cluster-aware routers with remote deployment. Otherwise Actors are created explicitly on a given node (this can be influenced using the configuration file in order to allow deployment decisions to be taken by the operations personnel and not the programmer).
+  
+### Programming Model
+
+#### Virtual Actors
+
+  * Orleans Grains are identified by their nominal type and a 128-bit GUID.
+  
+  * Akka Actors are identified by their ActorRef, which consists of the ActorPath (including physical network location and the names of all its ancestors) and its UID (a 32-bit integer that disambiguates different incarnations on the same ActorPath).
+
+Actors as well as Grains do not share memory with their peers, both only communicate via message-passing.
+
+##### Perpetual Existence
+
+discussed above
+
+##### Automatic Instantiation
+
+  * An Orleans Grain may at any given time have one or more activations (active instances) depending on deployment mode. This implies that stateful Grains will have to consider that there may be copies running outside of their control that are responsible for the same piece of data or functionality if automatic scale-out is enabled (see below).
```

Comment:
you can. But it also says: "responsible for the same piece of data ". Stateless actors are not responsible for data, they logically do not have state. Of course in practice they may have fields in memory and state, but there is no notion of "actor state" for those.